src/components/text-ellipsis/TextEllipsis.tsx

```diff
@@ -48,12 +48,16 @@ export const TextEllipsis: React.FC<TextEllipsisProps> = ({
       { children }
     </span>)
   }
+
+  if(!tooltipProps?.description && !children) {
```

Comment:
The description is a required prop on the Tooltip component, why we need to check it here?

Reply:
tooltipProps is `Partial<Partial<TooltipProps>` so any value could be undefined.

`TextEllipsis` works so that it will by default pass the `props.children` as the description for `Tooltip`, unless you override it with `tooltipProps.description`.

Instead of having to add checks for if your `children` or `tooltipProps.description` are undefined I think it's better if we can pass that to TextEllipsis and it handles it internally.